README.md

```diff
@@ -14,13 +14,13 @@ It's meant for ad-hoc querying of git repositories on disk through a common inte
 ## Installation
 
 ```
-go install -v -tags=sqlite_vtable github.com/augmentable-dev/gitqlite
+go get -v -tags=sqlite_vtable github.com/augmentable-dev/gitqlite
```

Comment:
Correct me if I'm wrong but I believe this will download gitqlite files to the GOPATH and will allow you to import it to other programs. But, to use the program the `go install ..`. command will still need to run for the gitqlite binary to be built and added to the gobin